src/ensembl/src/content/app/entity-viewer/shared/rest/rest-data-fetchers/proteinData.ts

```diff
@@ -87,3 +87,20 @@ export const fetchProteinSummaryStats = async (
 
   return restProteinSummaryAdaptor(proteinStatsData[xrefId]);
 };
+
+type InterproResponse = {
+  entries: unknown;
+};
+
+export const checkInterproUniprotId = async (
```

Comment:
I am wondering if `isInterproUniprotIdValid` is a better name. Not that I don't like this function's name.

Reply:
I think it will cause some confusion as we are only checking if the data is available for the given ID.